.github/workflows/linux_coverage.yml

```diff
@@ -21,6 +21,10 @@ jobs:
     - uses: actions/checkout@v2
     - name: Update apt repositories for ccache
       run: apt update
+    - name: Setup CMake
+      uses: jwlawson/actions-setup-cmake@v1.12
+      with:
+        cmake-version: '3.24.1'
```

Comment:
I don't think we need to add this here. The non-HIP CI image already has 3.22.1.